tests/legacy-cli/e2e/tests/misc/version.ts

```diff
@@ -1,13 +1,19 @@
-import {deleteFile} from '../../utils/fs';
-import {ng} from '../../utils/process';
-
-
-export default function() {
-  return ng('version')
-    .then(() => deleteFile('angular.json'))
-    // doesn't fail on a project with missing angular.json
-    .then(() => ng('version'))
-    // Doesn't fail outside a project.
-    .then(() => process.chdir('/'))
-    .then(() => ng('version'));
+import { deleteFile } from '../../utils/fs';
+import { ng } from '../../utils/process';
+
+
+export default async function() {
+  await ng('version');
```

Comment:
Could you take the stdout from this and verify it's the same as with `--version`.

Reply:
Done.